ncm-interactivelimits/src/main/pan/components/interactivelimits/config.pan

```diff
@@ -5,5 +5,18 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
+include { 'components/${project.artifactId}/schema' };
+include { 'pan/functions' };
+
+# Package to install.
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+# Set prefix to root of component configuration.
+prefix '/software/components/${project.artifactId}';
+
+#'version' = '${project.version}';
+#'package' = 'NCM::Component';
```

Comment:
Same as comment on #602 - these lines should either be removed a part of clean up or uncommented because they are required.